src/api/lib/tasks/dev/notifications.rake

```diff
@@ -21,7 +19,8 @@ namespace :dev do
       requestor_project = Project.find_by(name: 'requestor_project') || RakeSupport.create_and_assign_project('requestor_project', requestor)
 
       # Create notification for roles revoked
-      User.find_by(login: 'Iggy').run_as do
+      iggy = User.where(login: 'Iggy').first || create(:confirmed_user, login: 'Iggy', create_home_project: true)
+      iggy.run_as do
```

Comment:
That works on a fresh database.

But for a database where the `dev:test_data:create` has been run, the home project of the user Iggy is not created. See: https://github.com/openSUSE/open-build-service/blob/5a23b91410ef70ecf06e0aebc20f99e950f93791/src/api/lib/tasks/dev.rake#L103

We can create a new user or modify the creation of the the Iggy user to add `create_home_project: true` where I pointed to before.

Reply:
The home project for the user Iggy gets created later on, here: https://github.com/openSUSE/open-build-service/blob/5a23b91410ef70ecf06e0aebc20f99e950f93791/src/api/lib/tasks/dev.rake#L120

No need to change the code as I suggested. Sorry for the noise.